pipelines/cemba/cemba_methylcseq/CEMBA.wdl

```diff
@@ -1066,6 +1074,38 @@ task MethylationTypeCaller {
   }
 }
 
+# create a ALLC from VCF 
+task VCFtoALLC {
+    input {
+      File methylation_vcf_output_name
+    }
+
+  # input file size
+  Float input_size = 2*size(methylation_vcf_output_name, "GB") 
```

Comment:
This is confusing because it is more than the size of the input. I would remove this calculation and add the disk size calculation directly to the inputs block. Also if you can add the memory size to the inputs block, it allows more flexibility at run time 
```suggestion
```